README.md

```diff
@@ -43,7 +43,7 @@ The command above will display a list of available options. Make sure you read [
 
 ### Create a migration file with tables from your database
 
-To create your first migration file, execute:
+If you do not want to write your first migration file by hand, you can generate one using the [cakephp/bake](https://github.com/cakephp/bake) plugin:
```

Comment:
True, but the bake plugin is a suggestion of the migration plugin not a requirement, so if someone tried this it would fail.

Bake is included as a require-dev in the app though, so perhaps it would work, but I thought that telling people it was nothing to do with migrations would be valuable.

Perhaps it needs rewording?